LICENSE.txt

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2024 SAP SE or an SAP affiliate company.
```

Comment:
This is actually supposed to be a placeholder, so we should change it back as it is in the original license file (https://www.apache.org/licenses/LICENSE-2.0.txt)